backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  accessibilityId              String?                       @unique() @map("accessibility_id") @db.Uuid
+  demographicsId               String?                       @unique() @map("demographics_id") @db.Uuid
+  applicationFlaggedSet        ApplicationFlaggedSet[]
+  applicant                    Applicant?                    @relation(fields: [applicantId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  accessibility                Accessibility?                @relation(fields: [accessibilityId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  alternateContact             AlternateContact?             @relation(fields: [alternateContactId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicationsAlternateAddress Address?                      @relation("applications_alternate_address", fields: [alternateAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  userAccounts                 UserAccounts?                 @relation(fields: [userId], references: [id], onUpdate: NoAction)
+  applicationsMailingAddress   Address?                      @relation("applications_mailing_address", fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                     listings?                     @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  demographics                 Demographics?                 @relation(fields: [demographicsId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  preferredUnitTypes           UnitTypes[]
+  householdMember              HouseholdMember[]
+
+  @@unique([listingId, confirmationCode])
+  @@index([listingId])
+  @@map("applications")
+}
+
+// Note: [file_id, label] formerly max length 128
+model Assets {
+  id                            String              @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                     DateTime            @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                     DateTime            @updatedAt @map("updated_at") @db.Timestamp(6)
+  fileId                        String              @map("file_id")
+  label                         String
+  listingEvents                 ListingEvents[]
+  listingImages                 ListingImages[]
+  buildingSelectionCriteriaFile listings[]          @relation("building_selection_criteria_file")
+  listingsResult                listings[]          @relation("listings_result")
+  paperApplications             PaperApplications[]
+
+  @@map("assets")
+}
+
+// Note: [name] formerly max length 64
+model CronJob {
+  id          String    @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name        String?
+  lastRunDate DateTime? @map("last_run_date") @db.Timestamptz(6)
+  createdAt   DateTime  @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt   DateTime  @updatedAt @map("updated_at") @db.Timestamp(6)
```

Comment:
Wondering if these fields should also default to now

Reply:
good question! 
`@updatedAt` is prisma's way of handling this case (basically marks a column as an updatedAt column

so we don't need to manage defaults ourselves